Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -326,35 +327,98 @@ PhysicalParticleContainer::AddGaussianBeam (
 }
 
 void
-PhysicalParticleContainer::AddPlasmaFromFile (const std::string s_f, amrex::Real q_tot)
+PhysicalParticleContainer::AddPlasmaFromFile(const std::string s_f,
+                                               amrex::Real q_tot)
 {
 #ifdef WARPX_USE_OPENPMD
```

Comment:
Are we running this with one MPI rank only so far?

```suggestion
    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(
        ParallelDescriptor::NProcs() == 1,
        "AddPlasmaFromFile works only with one rank / serial WarpX for now\n");
        
#ifdef WARPX_USE_OPENPMD
```

Reply:
wait, that one is off! do not apply :)